frontend/src/templates/Field/Table/TableField.tsx

```diff
@@ -82,15 +89,15 @@ export const TableField = ({
         }
       }
     }
-  }, [append, fields.length, hasMinRowsChanged, remove, schema])
+  }, [appendTableRow, fields.length, hasMinRowsChanged, remove, schema])
 
   const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
     useTable({ columns: columnsData, data: fields })
 
   const handleAddRow = useCallback(() => {
     if (!schema.maximumRows || fields.length >= schema.maximumRows) return
-    return append(createTableRow(schema))
-  }, [append, fields.length, schema])
+    return appendTableRow()
```

Comment:
this is a regression for UX. Should we only prevent focus if in admin form builder preview?

Reply:
the problem occurs in public form tho

Reply:
Maybe there is a way to prevent triggering on blur when clicking the button. But this then goes to the same issue that we are having with verified field buttons if we don't want the validation to trigger on click. The difference is that there, respondents type the input first but here they don't input anything, so they wouldn't expect to be greeted with an error instantly.